src/ansys/geometry/core/designer/geometry_commands.py

```diff
@@ -1212,3 +1188,93 @@ def get_round_info(self, face: "Face") -> tuple[bool, Real]:
         result = self._commands_stub.GetRoundInfo(RoundInfoRequest(face=face._grpc_id))
 
         return (result.along_u, result.radius)
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def move_rotate(
+        self,
+        selection: Union["Body", list["Body"], "Face", list["Face"]],
+        axis: Line,
+        angle: Real,
+        options: MoveOptions = None,
+    ) -> bool:
+        """Rotate a selection around an axis for a given angle.
+
+        Parameters
+        ----------
+        selection : Body | list[Body] | Face | list[Face]
+            Selection to rotate.
+        axis : Line
+            Axis of rotation.
+        angle : Real
+            Angle of rotation in radians.
+        options : MoveOptions
+            Options for the rotation.
+
+        Returns
+        -------
+        bool:
+            ``True`` when successful, ``False`` when failed.
+        """
+        from ansys.geometry.core.designer.body import Body
+        from ansys.geometry.core.designer.face import Face
+
+        selection: list[Body | Face] = selection if isinstance(selection, list) else [selection]
+
+        check_type_all_elements_in_iterable(selection, (Body, Face))
+        check_is_float_int(angle, "angle")
+
+        result = self._commands_stub.MoveRotate(
+            MoveRotateRequest(
+                selection=[object._grpc_id for object in selection],
+                axis=line_to_grpc_line(axis),
+                angle=angle,
+            )
+        )
+
+        return result.success
+
+    @protect_grpc
+    @min_backend_version(25, 2, 0)
+    def move_translate(
+        self,
+        selection: Union["Body", list["Body"], "Face", list["Face"]],
+        direction: UnitVector3D,
+        distance: Real,
```

Comment:
Same comment as for Angle... We should use the Distance object and the Quantity objects as potential inputs.